deployment/ccip/changeset/cs_sync_usdc_domains_with_chains_test.go

```diff
@@ -0,0 +1,368 @@
+package changeset_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/testhelpers"
+	commonchangeset "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	commontypes "github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/deployment/environment/memory"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+func deployChainWithUSDCTokenPool(
```

Comment:
Do we not already have this in NewEnvironmentWithJobsAndContracts with Testconfig as USDC Deployment set to true?

Reply:
https://github.com/smartcontractkit/chainlink/pull/16062/commits/240fc25afb47237370875de60c6cdf621cc39bf0